src/other/utils.jl

```diff
@@ -181,6 +181,34 @@ end
 
 _findall(B) = findall(B)
 
+function _findall(B::AbstractVector{Bool})
```

Comment:
How about `function _findall(B::T) where {T<:AbstractVector{Bool}}` or something similar?

Reply:
This is exactly the same AFAICT.

Reply:
This is because tuples in Julia are covariant, see https://docs.julialang.org/en/v1/manual/types/#Tuple-Types

Reply:
Good to know!